benchmark/pom.xml

```diff
@@ -39,7 +39,7 @@
         <dependency>
             <groupId>io.prometheus</groupId>
             <artifactId>client</artifactId>
-            <version>0.0.11-SNAPSHOT</version>
+            <version>0.0.10</version>
```

Comment:
0.0.11 isn't released yet, so 0.0.10 is the last (ever) release of the original client.